### PR TITLE
Fallback if github_key.dest not available

### PR DESCRIPTION
**Pull request by vkurup**
One approach to address #20.

### CODE REVIEW

**Comment by vkurup** [comment]
@jbradberry This is updated.

File: tasks/main.yml [hunk 1]
```diff
@@ -71,9 +71,14 @@
   service: name=supervisor state=restarted
   when: supervisord_conf|changed
 
+- name: store path to github key
+  set_fact:
+    github_key_path: "{{ ssh_dir }}/github"
+  when: github_deploy_key|default('')
```
Comment (vkurup) [inline tasks/main.yml:77]: Actually do I need that `default` filter?

Comment (jbradberry) [reply tasks/main.yml:77]: I think so.  If I remember correctly, the `when` clause will throw an error if the variable is not defined without it.

**Comment by jbradberry** [comment]
:+1:

**Pull request merged**